### PR TITLE
fix(client): commitListItem className

### LINKED ISSUE #4211: 🐛 Bug Report: `CommitListItem` - width and padding problems
### What happened?

1. The padding on hover (the first item) and the selected item (the 4th item) should have the same width
2. the padding-right should be smaller
3. gap/padding/margin between the commit items
 
Screen Shot 2022-10-20 at 10 30 23

### What you expected to happen

It should look like in entity page - the fields list
Example on this issue: https://github.com/amplication/amplication/issues/3933 

### How to reproduce

create a project with a service
click on build and go to the commits page
click on build again to see the hover effect and width differences 

### Amplication version

1.0.3

### Environment

_No response_

### Are you willing to submit PR?

_No response_

### PR DESCRIPTION
**Pull request by chaiwattsw**
Close #4211 

## PR Details

fix innerTabLink to have padding when hover and add __row to commitListItem className to fix gap problem

## PR Checklist
- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Layout/InnerTabLink.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 @mixin active-link {
   background-color: var(--black10);
   color: var(--black100);
+  padding: var(--default-spacing-small);
```
Comment (abrl91) [inline packages/amplication-client/src/Layout/InnerTabLink.scss:6]: `InnerTabLink` is a component that is used in a lot of places. Are you sure that this change doesn't affect anywhere else?

Comment (chaiwattsw) [reply packages/amplication-client/src/Layout/InnerTabLink.scss:6]: Kinda sure here is screenshot with `commitListItem` and another place that use `InnerTabLink`

Screen Shot 2565-10-28 at 00 20 33
Screen Shot 2565-10-28 at 00 22 22

Comment (chaiwattsw) [reply packages/amplication-client/src/Layout/InnerTabLink.scss:6]: or we should add width: 100% to `.inner-tab-link` instead?

**Comment by abrl91** [comment]
run branch

**Comment by abrl91** [comment]
Thanks @chaiwattsw I will run your branch on my computer and review the changes

File: packages/amplication-client/src/VersionControl/CommitListItem.tsx [hunk 2]
```diff
@@ -18,12 +18,12 @@ export const CommitListItem = ({ commit, projectId }: Props) => {
...
   const { commitStatus } = useCommitStatus(commit);
   return (
-    <div className={CLASS_NAME}>
+    <div className={`${CLASS_NAME}__row`}>
       <InnerTabLink
         icon=""
         to={`/${currentWorkspace?.id}/${projectId}/commits/${commit.id}`}
       >
-        <div className={`${CLASS_NAME}`}>
+        <div className={CLASS_NAME}>
```
Comment (arielweinberger) [inline packages/amplication-client/src/VersionControl/CommitListItem.tsx:26]: Thanks for fixing this, the way the string was interpolated before indeed did not make sense.

Comment (chaiwattsw) [reply packages/amplication-client/src/VersionControl/CommitListItem.tsx:26]: You're welcome 😃

**Comment by chaiwattsw** [comment]
> Thanks @chaiwattsw I will run your branch on my computer and review the changes

Sure!

**Comment by abrl91** [comment]
@chaiwattsw have a look at this:

https://user-images.githubusercontent.com/39680385/198584395-446867f9-9c27-4a95-b92c-758b0d50c445.mov

The change affects the message broker setting page:

Another side effect is on the projects page and topics page. The right side of the `inner-tab-link` "lost" its border-radius:

Screen Shot 2022-10-28 at 15 34 34

Thank you for your time and effort, but we need a different solution.

**Comment by chaiwattsw** [comment]
> @chaiwattsw have a look at this:
> 
>  Screen.Recording.2022-10-28.at.15.12.24.mov 
> The change affects the message broker setting page:
> 
> Another side effect is on the projects page and topics page. The right side of the `inner-tab-link` "lost" its border-radius:
> 
> Screen Shot 2022-10-28 at 15 34 34
> 
> Thank you for your time and effort, but we need a different solution.

Please check again @abrl91 

Screen Shot 2565-10-28 at 20 19 13
Screen Shot 2565-10-28 at 20 20 03
Screen Shot 2565-10-28 at 20 21 39
Screen Shot 2565-10-28 at 20 36 12

**Comment by abrl91** [comment]
@chaiwattsw  there is a difference between a running commit and a non-running commit

image

**Comment by abrl91** [comment]
@chaiwattsw can you run in the root `npm run format` (to fix the CI)

**Comment by chaiwattsw** [comment]
> @chaiwattsw can you run in the root `npm run format` (to fix the CI)

@abrl91 fixed

**Comment by abrl91** [comment]
> @chaiwattsw there is a difference between a running commit and a non-running commit
> 
> image

@chaiwattsw what about this?

**Comment by chaiwattsw** [comment]
Hmmm not sure why CI failed

**Comment by souravjain540** [comment]
@abrl91  please review

**Comment by abrl91** [comment]
> @abrl91 please review

@chaiwattsw I will review it on Monday

**Review by abrl91 — APPROVED** [review]

**Pull request merged**